### DISCUSSION TITLE
Datastoring of uploads

### DISCUSSION THREAD

**Discussion by Kiilangelo** [openai/whisper#1381] [Q&A]
Hi! 
Does Whisper store and/or learn from the data you upload? 
If I use the AI-tool and upload an audiofile, is it then stored somewhere?

Thank you,

Best regards,
Anders

**Comment by ryanheise** [comment]
This GitHub repository is about the Python program called "Whisper", which has no upload feature, it runs entirely on your computer.

Perhaps you want to ask about one of OpenAI's APIs where you upload data to their servers and they process your data on their computers? If so, you might want to ask over at the OpenAI forums: https://community.openai.com/

**Reply by Kiilangelo** [reply]
Ah okay, cool. Thank you.
